trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py

```diff
@@ -159,16 +162,17 @@ def compute_airfoil_polars(a_geo,a_polar,use_pre_stall_data=True):
         CD_sur = RectBivariateSpline(airfoil_polar_data.reynolds_number[i],AoA_sweep_radians, CD[i,:,:])   
         
         CL_surs[a_geo[i]]  = CL_sur
-        CD_surs[a_geo[i]]  = CD_sur       
-      
+        CD_surs[a_geo[i]]  = CD_sur   
+        aoa_from_polars.append(airfoil_aoa)
+        
     airfoil_data.angle_of_attacks              = AoA_sweep_radians
     airfoil_data.lift_coefficient_surrogates   = CL_surs
     airfoil_data.drag_coefficient_surrogates   = CD_surs 
     
     airfoil_data.lift_coefficients_from_polar  = airfoil_polar_data.lift_coefficients
     airfoil_data.drag_coefficients_from_polar  = airfoil_polar_data.drag_coefficients
     airfoil_data.re_from_polar  = airfoil_polar_data.reynolds_number
-    airfoil_data.aoa_from_polar = airfoil_polar_data.angle_of_attacks
+    airfoil_data.aoa_from_polar = np.asarray(aoa_from_polars) # airfoil_polar_data.angle_of_attacks
```

Comment:
have you tried to import several polars of different dimensions. That was the reason I left them as lists?

Reply:
good catch, I will change this line back. The reason I had this was because it made plotting the polars easier to have the same numbers, but you are right.